tests/zip/BUILD

```diff
@@ -206,8 +211,9 @@ py_test(
         # it with this name.
         "test_zip_out.foo",
         "test_zip_package_dir0.zip",
-        "test_zip_timestamp.zip",
         "test_zip_permissions.zip",
+        "test_zip_timestamp.zip",
+        "test_zip_tree",
```

Comment:
Any particular reason why this one doesn't end with `.zip` and the others do?

Reply:
No, just an oversight. Fixed.